release/models/aft/openconfig-aft-common.yang

```diff
@@ -244,11 +258,156 @@ submodule openconfig-aft-common {
           }
         }
 
+        container encap-headers {
+          description
+            "Container for packet encapsulation headers.  When present, this
+            container indicates that encapsulation of the packet matching the
+            next-hop is performed using a stack of one or more packets defined
+            in the list encap-header.
+
+            Each entry in the list must indicate an encapsulation type and
+            populate a container with the parameters for that encapsulation
+            header.";
+
+          list encap-header {
+            description
+              "A list of headers added on top of a packet.  The first entry
+              in the list is the inner-most packet at the bottom of the
```

Comment:
"at the bottom of the stack" -- can we say "the inner-most is closer to the original packet"?

Reply:
done